eng/pipelines/runtime-perf-jobs.yml

```diff
@@ -148,122 +134,123 @@ jobs:
         liveLibrariesBuildConfig: Release
-        runKind: micro
+        runtimeType: mono
+        codeGenType: 'AOT'
+        runKind: micro_mono
         logicalMachine: 'perftiger'
-        pgoRunType: nodynamicpgo
-        additionalJobIdentifier: nodynamicpgo
         runtimeRepoAlias: ${{ parameters.runtimeRepoAlias }}
         performanceRepoAlias: ${{ parameters.performanceRepoAlias }}
         ${{ each parameter in parameters.jobParameters }}:
           ${{ parameter.key }}: ${{ parameter.value }}
 
+  # run mono aot microbenchmarks perf job - perfviper
   - template: /eng/pipelines/common/platform-matrix.yml@${{ parameters.runtimeRepoAlias }}
     parameters:
-      jobTemplate: /eng/pipelines/templates/runtime-perf-job.yml@${{ parameters.performanceRepoAlias }}
+      jobTemplate: /eng/pipelines/templates/runtime-perf-job.yml@${{ parameters.performanceRepoAlias }} # NOTE: should we move this file out of coreclr tempelates because it contains mono jobs?
       buildConfig: release
-      runtimeFlavor: coreclr
+      runtimeFlavor: aot
       platforms:
       - linux_x64
       jobParameters:
         liveLibrariesBuildConfig: Release
-        runKind: micro
-        logicalMachine: 'perftiger'
-        pgoRunType: nodynamicpgo
-        additionalJobIdentifier: nodynamicpgo
+        runtimeType: mono
+        codeGenType: 'AOT'
+        runKind: micro_mono
+        logicalMachine: 'perfviper'
         runtimeRepoAlias: ${{ parameters.runtimeRepoAlias }}
         performanceRepoAlias: ${{ parameters.performanceRepoAlias }}
         ${{ each parameter in parameters.jobParameters }}:
           ${{ parameter.key }}: ${{ parameter.value }}
 
-  # run coreclr perftiger microbenchmarks no R2R perf jobs
+  # run coreclr perftiger microbenchmarks perf job
   - template: /eng/pipelines/common/platform-matrix.yml@${{ parameters.runtimeRepoAlias }}
     parameters:
       jobTemplate: /eng/pipelines/templates/runtime-perf-job.yml@${{ parameters.performanceRepoAlias }}
       buildConfig: release
       runtimeFlavor: coreclr
       platforms:
+      - linux_x64
       - windows_x64
+      - windows_x86
+      - linux_musl_x64
       jobParameters:
         liveLibrariesBuildConfig: Release
         runKind: micro
         logicalMachine: 'perftiger'
-        r2rRunType: nor2r
-        additionalJobIdentifier: nor2r
         runtimeRepoAlias: ${{ parameters.runtimeRepoAlias }}
         performanceRepoAlias: ${{ parameters.performanceRepoAlias }}
         ${{ each parameter in parameters.jobParameters }}:
           ${{ parameter.key }}: ${{ parameter.value }}
 
+  # run coreclr perfviper microbenchmarks perf job
   - template: /eng/pipelines/common/platform-matrix.yml@${{ parameters.runtimeRepoAlias }}
     parameters:
       jobTemplate: /eng/pipelines/templates/runtime-perf-job.yml@${{ parameters.performanceRepoAlias }}
       buildConfig: release
       runtimeFlavor: coreclr
       platforms:
       - linux_x64
+      - windows_x64
```

Comment:
On line 277 it seems like we already have a definition for this job. We probably only need one.